bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+use Zend\Console;
+use Zend\Loader\StandardAutoloader;
```

Comment:
Which brings me to an interesting dilemma... The script makes use of `Zend\Console\Getopt`, which would mean that zend-view would need to add zend-console as a dependency, which I doubt we want to do.

I think we could likely simplify the script to have the signature:

```bash
$ templatemap_generator.php <directory>
```

and have it spit what it generates to STDOUT; developers can then pipe that to a file. As an example, to overwrite:

```bash
$ templatemap_generator.php module/Album/view > config/autoload/views.global.php
```

To append:

```bash
$ templatemap_generator.php module/Album/view >> config/autoload/views.global.php
```

Alternately, we could require that they pass the full set of files via globbing, which would accommodate the current `--extensions` argument:

```bash
$ templatemap_generator.php module/Album/view/**/*.phtml >> module/Album/config/template_map.php
```

If the script assumes that the map should be relative to the `PWD`, then we could suggest this workflow:

```bash
$ cd module/Album/view
$ ../../../vendor/bin/templatemap_generator.php ./**/*.phtml > ../config/template_map.php
```

Calling the script with no arguments or with `--help` or `-h` would provide the usage message.

The above essentially accommodates all current options, and would allow eliminating the dependency on zend-console; we'd only have to parse `$argv` and grab the first argument.

Thoughts?